tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCheckCLDR.java

```diff
@@ -476,8 +478,10 @@ public void TestAllLocales() {
         LanguageTagParser ltp = new LanguageTagParser();
         Set<String> locales = new HashSet<>();
         for (String locale : getInclusion() <= 5 ? eightPointLocales : factory.getAvailable()) {
+            checkNullWithInheritanceMark(locale);
```

Comment:
Actually I've just realized, this doesn't really avoid adding a cycle through the locales, since the original test waits until after this loop to cycle with `checkLocale`

Given the hurry to get this ready, I suggest merging this anyway and following up to address the performance